src/global.d.ts

```diff
@@ -0,0 +1,15 @@
+import "reactn";
+
+/*
+  This sets the shape of our global state since we want our global
+  state to be typed. Should be the same shape as the global object
+  initialized in src/pages/index.tsx
+*/
+declare module "reactn/default" {
+  export interface State {
+    tutorialName: string;
+    tutorialStep: number;
+    selectorDisplay: string;

```

Comment:
You've probably already seen my comment for this but I'll post it again for reference.

> Personally, I think that `selectorDisplay` and `tutorialDisplay` are better being boolean values, and then in the styled components you can state what CSS property they have based on the boolean value. Better to keep CSS properties within css in many cases.